api/jobs/updateGeoIp.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const job = require('../parts/jobs/job.js'),
+    log = require('../utils/log.js')('job:updateGeoIp'),
+    { exec } = require('child_process');
+
+/** Class for the job of updating geo-ip's **/
+class UpdateGeoIP extends job.Job {
+    /**
+     * Run the update geo-ip job
+     * @param {done} done callback
+     */
+    run(done) {
+        log("Updating geo-ip's..");
+        exec("countly update geoip", (err) => {
+            if (err) {
+                log("Error occurred while updating geoip:" + err);
+                done();
+            }
+            log("Updating devices...");
+            exec("countly update devices", (err) => {
+                if (err) {
+                    log("Error occurred while updating devices:" + err);
+                }
+                done();
+            });
+        });
+    }
```

Comment:
in this whole scenario, callback(done) is being called twice
1. if error occurs
2. after countly update device

is this desired?

Reply:
point being, incase of error, after the done callback is called,
the countly update devices command will run anyway.